src/test/java/io/reactivex/plugins/RxJavaPluginsTest.java

```diff
@@ -16,14 +16,26 @@
 
 package io.reactivex.plugins;
 
-import io.reactivex.*;
+import io.reactivex.Completable;
```

Comment:
Yes. Changed the code style settings accordingly.
Do you have a code style configuration for IntelliJ Idea?